docs/source/traits_user_manual/defining.rst

```diff
@@ -278,7 +278,9 @@ the table.
 +------------------+----------------------------------------------------------+
 | Name             | Callable Signature                                       |
 +==================+==========================================================+
-| Any              | Any( [*value* = None, \*\*\ *metadata*] )                |
```

Comment:
Corrected the name of the initial parameter from `value` to `default_value` (though I expect few people will be passing that parameter by name).